Sources/MCPToolMacrosPlugin/MCPToolMacrosPlugin.swift

```diff
@@ -0,0 +1,7 @@
+// Re-export the macro definition and related utilities
+@_exported import MCP
```

Comment:
`@_exported import MCP` widens the public surface for dependents. If the goal is convenience for examples, fine; otherwise consider a normal `import MCP` in consumers to keep the plugin module’s exports minimal.